pages/linux/ffuf.md

```diff
@@ -0,0 +1,28 @@
+# ffuf
+
+> Subdomain and directory discovery tool.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Discover directories using a wordlist on a target website colorized and verbose ouput:
```

Comment:
I think we can highlight the argument meaning in the example description with the `[]`:
```suggestion
- Discover directories using a [w]ordlist on a target [u]rl with [c]olorized and [v]erbose ouput:
```
@marchersimon wdyt?

Reply:
I'm not against it, but I think that would be a paradigm shift for the TLDR project, as it is not currently implemented in other TLDR pages, i've come across.

Reply:
@otterbaub we use there in some pages like `7z` and `tar` where we don't have long argument options and it's an easy way to let the user know what is each flag.

Reply:
Good enough for me, Convinced.

Addressed in a4924d75148c45de768d9fee1e6b4d83d36560ce